contracts/governance/Vesting/fouryear/FourYearVestingLogic.sol

```diff
@@ -137,6 +138,42 @@ contract FourYearVestingLogic is IFourYearVesting, FourYearVestingStorage, Appro
 		emit DividendsCollected(msg.sender, _loanPoolToken, _receiver, _maxCheckpoints);
 	}
 
+	/**
+	 * @notice Change token owner - only vesting owner.
+	 * @dev Modifies token owner. This must be followed by approval
+	 * from token owner.
+	 * @param _newTokenOwner Address of new token owner.
+	 * */
+	function changeTokenOwner(address _newTokenOwner) public onlyOwner {
```

Comment:
auditors will yell: EXTERNAL! ) on all public

Reply:
Changed to external